### PR TITLE
Update Read the Docs Python version to 3.8

### LINKED ISSUE #32: Read the Docs not building correctly
Take a look at https://readthedocs.org/projects/adafruit-circuitpython-funhouse/builds/15808326/ 

Fourth step uses python3.7. Results in same problem as fixed with #31.

Towards the end of the build run this error appears:

> NameError: name 'SocketType' is not defined

### PR DESCRIPTION
**Pull request by makermelissa**
Fixes #32.

### CODE REVIEW

**Review by ladyada — APPROVED** [review]

File: .readthedocs.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 version: 2
 
 python:
-  version: "3.7"
+  version: "3.8"
```
Comment (kattni) [inline .readthedocs.yaml:12]: This should be `"3.x"` to match what we're doing across all the libraries.

File: docs/conf.py [hunk 2]
```diff
@@ -37,7 +37,7 @@
 ]
 
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3.4", None),
+    "python": ("https://docs.python.org/3.8", None),
```
Comment (kattni) [inline docs/conf.py:40]: Should be `"https://docs.python.org/3.x"` to match libraries.

**Review by kattni — COMMENTED** [review]
Thanks for the fix! Suggested changes below.

**Comment by kattni** [comment]
Didn't see that this was already merged before posting changes.

**Pull request merged**